### ISSUE TITLE
Keyboard Help Dialog - needs updating based on ineraction changes

### ISSUE THREAD

**Issue by terracoda** [phetsims/molecules-and-light#303]
Current Dialog has slider instructions, so we need to update it, but I am not exactly sure what is be needed.
![Screen Shot 2020-03-12 at 3 21 03 PM](https://user-images.githubusercontent.com/1902116/76550463-23a21f00-6475-11ea-8d82-cc4cb3dbb5ae.png)

**Comment by terracoda** [comment]
The main interaction issue in the sim has been the Play, Pause and Step buttons.
I am wan to explore an HTML code change in #299

**Comment by arouinfar** [comment]
Good catch @terracoda. We should eliminate the entire "Light Source Slider" section of this dialog since it's now irrelevant.

>  The main interaction issue in the sim has been the Play, Pause and Step buttons.

Since these are all buttons, I don't think there's necessarily any special instructions we need to include in the keyboard dialog for these controls. Actually, I can't think of any keyboard actions that we need to describe in the dialog that don't already fall under "Basic Actions".

**Comment by terracoda** [comment]
@arouinfar I agree the interaction technically falls under "Basic Actions", but we still might want to describe the most important interaction?  It could help some students less familiar with keyboard interactions. 

I drafted some new content for consideration that we can discuss at Tuesday's design meeting.

**Comment by terracoda** [comment]
@arouinfar and @Matthew-Moore240, we didn't get to this item in the agenda in our last meeting.
I think this content would work nicely, for the keyboard shortcuts dialog. 

Do you both agree?

Note that content in square brackets represents key icons for the visual content which is listed first.

VISUAL CONTENT
**Keyboard Shortcuts**
[Tab] to get started

**Light Source**
Turn light source on or off [Enter] or [Space]

**Basic Actions**
Move to next item or group of items [Tab]
Move to previous item or group of items [Shift] + [Tab] 
Move between group items [Left Right] or [Up Down]
Press buttons [Space] or [Enter]
Exit a dialog [Esc]

NON-VISUAL CONTENT
H1: **Keyboard Shortcuts**
Tab to get started.

H2: **Light Source**
Turn light source on or off with Space or Enter keys.

H2: **Basic Actions**
Move to next item or group with Tab key.
Move to previous item or group with Shift plus Tab key.
Move between items in a group with the left and right arrow keys or the up and down arrow keys.
Press buttons with Space key.
Exit this dialog with Escape key.

**Comment by arouinfar** [comment]
@terracoda content looks pretty good, just a few comments.

> **Light Source**
Turn light source on or off [Enter] or [Space]

Should we swap the order to `Space` or `Enter` to match the order used for buttons under "Basic Actions" and the non-visual content for the Light Source?

> Press buttons with Space key.

Is there a reason to omit the `Enter` key here? This description doesn't line up with the visual content. We also mention both the `Space` and `Enter` keys in the non-visual content for the Light Source, why not here too?

**Comment by terracoda** [comment]
@arouinfar, good catch. It should be the same as other sims like Friction and BASE.

**Comment by terracoda** [comment]
I just checked BASE
![Screen Shot 2020-03-25 at 7 15 18 PM](https://user-images.githubusercontent.com/1902116/77588525-ff126200-6ecc-11ea-8f44-014d94f429ad.png)

**Comment by terracoda** [comment]
@jessegreenberg, would need a mock up for this?

**Comment by jessegreenberg** [comment]
Nope, thanks @terracoda, https://github.com/phetsims/molecules-and-light/issues/303#issuecomment-604088541 should be sufficient.

**Comment by terracoda** [comment]
@arouinfar, re the content under Basics Actions, it should be both [Space] and [Enter], but we left off [Enter] for simplicity.

We can keep that simple for now, or make an issue in common code to update it to include both keys. I forgot that it was simplified.

**Comment by terracoda** [comment]
@jessegreenberg and @arouinfar, I updated the content in https://github.com/phetsims/molecules-and-light/issues/303#issuecomment-604088541

Apologies for reading @arouinfar's entire comment :-)

**Comment by jessegreenberg** [comment]
Discussed during meeting today, this is ready!

**Comment by jessegreenberg** [comment]
UPdated, here is the dialog now:
![image](https://user-images.githubusercontent.com/6396244/79169295-118c0700-7dba-11ea-979b-8d0312a15868.png)

And the PDOm content:
![image](https://user-images.githubusercontent.com/6396244/79169310-181a7e80-7dba-11ea-982b-f78b28bc3dc4.png)

Reassigning for review.

**Comment by terracoda** [comment]
@jessegreenberg, the Space key needs to come first in the visual representation like in https://github.com/phetsims/molecules-and-light/issues/303#issuecomment-604104539

Apologies for the original confusion in the Design document.

**Comment by jessegreenberg** [comment]
@terracoda the "Enter or Space" icon is a reusable component, also used in sims like Molarity
![image](https://user-images.githubusercontent.com/6396244/79235276-658a0080-7e39-11ea-82af-1cace69e983e.png)

If I change the order here the order will also change for molarity, is that OK?

**Comment by terracoda** [comment]
Yes, that should be just fine. I didn't look carefully at other sims when I  provided Molarity's content.

Will the change automatically fix the accessible text, too, in Molarity, or do I need to make an issue?

**Comment by terracoda** [comment]
@jessegreenberg, by the way, nice catch!

**Comment by jessegreenberg** [comment]
OK, thanks @terracoda , the order has been switched.

**Comment by terracoda** [comment]
Looks good, @jessegreenberg. Thanks and closing.
